docs/reference/experiment-config.txt

```diff
@@ -1185,6 +1185,8 @@ The table below describes the field names that have changed.
     - min_validation_period
   * - min_checkpoint_period
     - min_checkpoint_period
+  * - batches_per_step
+    - scheduling_unit
```

Comment:
This was intentionally not included:
> For each configuration setting in the table, if it exists in your configuration, it should be
renamed as shown above and changed to use the new units. To migrate the value to the new units,
renamed as shown above and changed to use the new units.

This table was for fields that changed from steps to "Training Units". Maybe make that clearer in the sentence above it and add ``scheduling_unit`` elsewhere more clearly (it's mentioned at the bottom, just as needing a rename)?

Reply:
Or just note in the table somehow that scheduling_units's units didn't change. Not sure what's best.

Reply:
that makes sense - add an additional column in the table; hopefully it's not overkill.